### ISSUE TITLE
What's happening with my Tabs?

### ISSUE THREAD

**Issue by ryexley** [ryexley/billy#1]
I'm trying to use the [radix-ui Tabs component](https://www.radix-ui.com/docs/primitives/components/tabs) to create a Login/Sign Up form for this app. As you can see in the [`tab-set.js` file](https://github.com/ryexley/billy/blob/feature/auth/app/components/tab-set.js#L60-L71) I've styled the `TabsPrimitive.Content` component, but [where I've imported and rendered that into my login page](https://github.com/ryexley/billy/blob/feature/auth/app/routes/login.jsx#L103-L142), it's not rendering with the styles. I'm not sure what I'm missing here, but I've looked over my code here several times, and am not seeing what I'm doing wrong. The thing that seems most strange to me is that I can see that a style class is getting applied to it, but when I look in browser dev tools, I don't see any of the styles for that class getting rendered, and I'm not sure why. It even seems to have overridden the styles that I've defined for my [text inputs](https://github.com/ryexley/billy/blob/feature/auth/app/components/text-input.jsx) and [button](https://github.com/ryexley/billy/blob/feature/auth/app/components/button.jsx) as well. Anyone have any ideas what I've missed here?

![image](https://user-images.githubusercontent.com/173844/144612114-84e1846e-8995-449f-93c0-2dab2eaf002a.png)

<img width="1278" alt="Screen Shot 2021-12-03 at 8 43 49 AM" src="https://user-images.githubusercontent.com/173844/144612690-cd353a63-ee24-4fb1-83a9-ccbd1cfe253f.png">

**Comment by peduarte** [comment]
Hi, would you mind trying it with the latest version of Stitches, which is `1.2.6`? Thanks!

**Comment by ryexley** [comment]
@peduarte I updated to `1.2.6`, and made the changes that @jjenzz [recommended in Discord](https://discord.com/channels/752614004387610674/803656530259738674/916362102992678974), and that does seem to have fixed the issue I was seeing. Her recommendation actually fixed it _before_ I upgraded, but I've upgraded as well. I _am_ still seeing the flicker [that she mentioned](https://discord.com/channels/752614004387610674/803656530259738674/916362727369359400) though, but I'm at least seeing the styling that I am expecting, which is ultimately where I needed to be. Thank you both for helping me get this solved!
